src/server/routes/event.js

```
@@ -10,6 +11,7 @@ function eventsHandler(logger)
 {
     router.post('/callback/:status', (req, res) =>
     {
+        rabbitEvents.emit(req.body.mj_message_id || req.body.MessageID, req.body.Email || req.body.email, req.params.status);
```

Change the order: do events.emit first, then mqEvents.emit
Reasoning: the events is known stable, mqEvents is new and we might have issues.
In any case we don't wan't stray issues impact prod as here clearly if mqEvents fails hard the other one might not be called at all.